README.md

```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
 
 # Build from source
 
+Before build make sure that **CGO_ENABLED** env is set to **1**
+
+```
+set CGO_ENABLED 1
```

Comment:
remove this

Reply:
Fixed. The set I think work with normal cmd